### PR TITLE
only get pasta eyeball if it's worth it

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

File: src/mood.ts [hunk 1]
```diff
@@ -73,8 +79,23 @@ export function meatMood(urKels = false, embezzlers = false): Mood {
...
-
+  if (myClass() !== $class`Pastamancer`) {
+    const isWorthIt =
+      (baseMeat * getModifier("Meat Drop", $effect`Pasta Eyeball`)) /
+        100 /
+        mpCost($skill`Bind Lasagmbie`) >
+      Math.max(
+        (1.5 * myLevel() + 5) / 80,
+        (2.5 * myLevel()) / mallPrice($item`generic mana potion`)
+      );
```
Comment (phulin) [inline src/mood.ts:90]: can you break this into sub-computations with names? A little incomprehensible right now.

File: src/mood.ts [hunk 2]
```diff
@@ -73,8 +79,23 @@ export function meatMood(urKels = false, embezzlers = false): Mood {
...
+    if (
+      isWorthIt ||
+      (have($item`porquoise-handled sixgun`) && monsterManuelAvailable()) ||
+      getFloristPlants()[$location`Barf Mountain`.toString()]?.includes("Pitcher Plant") ||
+      get("_bittycar") === "soulcar"
+    )
+      mood.skill($skill`Bind Lasagmbie`);
```
Comment (phulin) [inline src/mood.ts:97]: we should probably also apply to Spirit of Taking

File: src/mood.ts (hunk 1 above)
Comment (horrible-little-slime) [reply src/mood.ts:90]: sure!

**Comment by horrible-little-slime** [comment]
Closing this PR after MP changes; will keep the branch alive in case we change our minds.

**Pull request closed without merging**